datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -481,21 +481,21 @@ def metadata(check, check_duplicates, show_warnings):
                     (echo_failure, f"{current_check}:{line} interval should be an int, found '{row['interval']}'.")
                 )
 
+            if 'curated_metric' in row and row['curated_metric']:
+                for curated_metric_type in row['curated_metric'].split('|'):
+                    if curated_metric_type not in VALID_CURATED_METRIC_TYPES:
+                        errors = True
+                        display_queue.append(
+                            (
+                                echo_failure,
+                                f"{current_check}:{line} `{row['metric_name']}` contains invalid curated metric type.",
```

Comment:
```suggestion
                                f"{current_check}:{line} `{row['metric_name']}` contains invalid curated metric type: {curated_metric_type}",
```
let's also update the error message